troubleshoot/observability/troubleshoot-your-universal-profiling-agent-deployment.md

```diff
@@ -2,6 +2,10 @@
 navigation_title: "Universal Profiling"
 mapped_pages:
   - https://www.elastic.co/guide/en/observability/current/profiling-troubleshooting.html
+applies_to:
+  stack: all
+  serverless:
+    observability: all
```

Comment:
Last I heard Universal Profiling was not available in serverless. @elastic/obs-docs can anyone confirm? 

```suggestion
```

Reply:
That's correct. It's not currently in Serverless.